src/Tribe/Views/V2/Template/Title.php

```diff
@@ -125,8 +126,19 @@ protected function get_taxonomies(): array {
 	 * @return string The page title.
 	 */
 	public function build_title( $current_title = '', $depth = true, $sep = null ) {
-		$context = $this->context ?: tribe_context();
-		$posts   = $this->get_posts();
+		$context            = $this->context ?: tribe_context();
+		$event_display_mode = $context->get( 'event_display_mode' );
+
+		// If the slug is `default`, get the slug another way.
+		if ( 'default' === $event_display_mode ) {
+			$manager      = tribe( Manager::class );
+			$event_display_mode  = $manager->get_default_view_option();
+		}
+
+		$posts = [];
+		if ( $event_display_mode !== 'month' && $event_display_mode !== 'day' ) {
+			$posts = $this->get_posts();
+		}
```

Comment:
I added a conditional here to only get posts for list views. 

This also fixes the issue found in:
https://stellarwp.atlassian.net/browse/TEC-4937

I made this change as get_posts() does not need to run for the title for Month and Day view. 
There would be a corresponding change in Pro for the Week and All views.

Reply:
Here we can check: `trait List_Behavior`, if has that trait it's safe to assume it needs the `$this->get_posts()`.

Reply:
Ignore me, we dont have the `$view` instance at this point.